### PR TITLE
Adding more info around static nodes

### LINKED ISSUE #1461: bootnodes vs static nodes - add to explanation
https://besu.hyperledger.org/development/public-networks/how-to/connect/static-nodes

Suggest to expand the note on this page to highlight the important difference between bootnodes and static nodes -

Static nodes and bootnodes are only treated the same at startup (ie besu tries to connect to them all). However only static nodes (and enodes added via admin_addPeer) are included in "maintained peers" (to which besu regularly attempts to reconnect), not bootnodes. So if you only have bootnodes, after startup you're relying solely on discovery. So in small node networks, we would recommend static nodes (or both) to mitigate low peer count issues.

### PR DESCRIPTION
**Pull request by MadelineMurray**
Fixes #1461

### CODE REVIEW

**Comment by MadelineMurray** [comment]
@macfarla - I can't assign reviewers so I'm tagging you instead.

File: docs/public-networks/how-to/connect/static-nodes.md [hunk 1]
```diff
@@ -11,11 +11,15 @@ tags:
...
+Besu periodically initiates a connection to any unconnected static node. To mitigate low peer count issues in small networks, we recommend using static nodes, or static nodes and bootnodes. 
 
 :::tip
 
-Bootnodes and static nodes are parallel methods for finding peers. Depending on your use case, you can use only bootnodes, only static nodes, or both bootnodes and static nodes. For example, you run multiple nodes on Mainnet (discovery using bootnodes), but want to ensure your nodes are always connected (using static nodes).
+Bootnodes and static nodes are both methods for finding peers. Depending on your use case, you can use only bootnodes, only static nodes, or both bootnodes and static nodes. 
+
+For example:
+* You run multiple nodes on Mainnet, using bootnodes for discovery, but want to ensure your nodes are always connected, use static nodes.
+* You run a small network and want the nodes to reconnect if disconnected, use static nodes.
```
Comment (macfarla) [inline docs/public-networks/how-to/connect/static-nodes.md:22]:
```suggestion
* You run a small network and want your nodes to reconnect if disconnected, use static nodes. 
```

File: docs/public-networks/how-to/connect/static-nodes.md [hunk 2]
```diff
@@ -11,11 +11,15 @@ tags:
...
-Besu attempts to maintain connections with static nodes by periodically initiating a connection to any unconnected static node.
+Besu periodically initiates a connection to any unconnected static node. To mitigate low peer count issues in small networks, we recommend using static nodes, or static nodes and bootnodes. 
 
 :::tip
 
-Bootnodes and static nodes are parallel methods for finding peers. Depending on your use case, you can use only bootnodes, only static nodes, or both bootnodes and static nodes. For example, you run multiple nodes on Mainnet (discovery using bootnodes), but want to ensure your nodes are always connected (using static nodes).
+Bootnodes and static nodes are both methods for finding peers. Depending on your use case, you can use only bootnodes, only static nodes, or both bootnodes and static nodes. 
+
+For example:
+* You run multiple nodes on Mainnet, using bootnodes for discovery, but want to ensure your nodes are always connected, use static nodes.
```
Comment (macfarla) [inline docs/public-networks/how-to/connect/static-nodes.md:21]:
```suggestion
* You run multiple nodes on Mainnet, using bootnodes for discovery, but want to ensure your nodes are always connected to each other, use static nodes.
```

**Review by macfarla — APPROVED** [review]

**Comment by macfarla** [comment]
> @macfarla - I can't assign reviewers so I'm tagging you instead.

we could give you this permission

File: docs/public-networks/how-to/connect/static-nodes.md [hunk 3]
```diff
@@ -11,11 +11,15 @@ tags:
...
+Besu periodically initiates a connection to any unconnected static node. To mitigate low peer count issues in small networks, we recommend using static nodes, or static nodes and bootnodes. 
 
 :::tip
 
-Bootnodes and static nodes are parallel methods for finding peers. Depending on your use case, you can use only bootnodes, only static nodes, or both bootnodes and static nodes. For example, you run multiple nodes on Mainnet (discovery using bootnodes), but want to ensure your nodes are always connected (using static nodes).
+Bootnodes and static nodes are both methods for finding peers. Depending on your use case, you can use only bootnodes, only static nodes, or both bootnodes and static nodes. 
+
+For example:
+* You run multiple nodes on Mainnet, using bootnodes for discovery, but want to ensure your nodes are always connected to each other, use static nodes.
+* You run a small network and want your nodes to reconnect if disconnected, use static nodes.
```
Comment (alexandratran) [inline docs/public-networks/how-to/connect/static-nodes.md:22]:
```suggestion
* You run multiple nodes on Mainnet, using bootnodes for discovery, but want to ensure your nodes are always connected to each other, using static nodes.
* You run a small network and want your nodes to reconnect if disconnected, using static nodes. 
```

**Review by alexandratran — APPROVED** [review]

**Pull request merged**